internal/core/graphics/image.rs

```diff
@@ -473,6 +479,9 @@ impl ImageInner {
                                 });
                                 slice.fill_with(|| iter.next().unwrap());
                             }
+                            PixelFormat::SignedDistanceField => {
```

Comment:
Do you know where this might be called from?

Reply:
Currently this is not called, but could be called if we were to generate distance field for @image-url